db/schema.rb

```diff
@@ -815,12 +815,12 @@
   create_table "spree_shipments", force: :cascade do |t|
     t.string   "tracking",          limit: 255
     t.string   "number",            limit: 255
-    t.decimal  "cost",                          precision: 10, scale: 2
+    t.decimal  "cost",                          precision: 10, scale: 2, default: 0.0, null: false
```

Comment:
It feels great to gradually see more constraints making it into the DB. It feels like we're on the right path :heart_eyes: